controllers/applications.js

```diff
@@ -34,6 +34,30 @@ const createApplication = (req, res) => {
   });
 };
 
+const updateApplication = (req, res) => {
+  Applications.findByIdAndUpdate(
+    req.params.id,
+    {
+      positions: req.body.positions,
+      postulants: req.body.postulants,
+      interview: req.body.interview,
+      result: req.body.result,
+    },
+    (err, updatedApplication) => {
```

Comment:
add `{ new: true },` as the second params to return the updated application in the callback